setup.py

```diff
@@ -46,6 +46,7 @@
         'pytz',             # in anaconda
         'ruamel.yaml>=0.15.0',      # in anaconda
         'scikit-learn~=0.21.0',  # See PEP 440, compatible releases
+        'tables>=3.6'       # in anaconda
```

Comment:
Do we need something that is specifically in 3.6?

Reply:
Dont think so. It's just the version in my enviroment and I dint perform tests on other versions.

Reply:
Setup.py should list the minimum required version. 3.6 is the most recent one. I don't mind requiring it if we have to, but if not, we should maybe just require >= 3

Reply:
I'll test it.